control-plane/csi-driver/src/bin/node/filesystem_vol.rs

```diff
@@ -20,11 +20,15 @@ macro_rules! failure {
     (Code::$code:ident, $fmt:literal $(,$args:expr)+) => {{ let message = format!($fmt $(,$args)+); error!("{}", message); Status::new(Code::$code, message) }};
 }
 
+/// `nouuid` mount flag, to allow duplicate fs uuid.
+const NO_UUID_FLAG: &str = "nouuid";
+
 pub(crate) async fn stage_fs_volume(
     msg: &NodeStageVolumeRequest,
     device_path: &str,
     mnt: &MountVolume,
     filesystems: &[FileSystem],
+    clone_uuid: Option<&String>,
```

Comment:
Should this be a uuid rather than String?

Reply:
Yeah could be, but since comes from the publish context, we would need a round of conversion to uuid. That's why kept as string.

Reply:
At least should validate that it's a uuid before calling into fs utility

Reply:
Okay, sure. Would make that change.